po4a-update-templates.sh

```diff
@@ -52,9 +52,9 @@ do
     path="${dirname#$SRC_DIR/}"
 
     if [ "$dirname" = "$SRC_DIR" ]; then
-        potname=$basename.pot
+        potname="$basename".pot
```

Comment:
These two need the double quotes, as you don't know if the path will contain a space, which would otherwise cause the shell some problems...